solana/program_v2/package.json

```diff
@@ -38,8 +31,7 @@
     "mocha": "^10.0.0",
     "prettier": "^2.7.1",
     "ts-mocha": "^10.0.0",
-    "typescript": "^4.8.3",
-    "husky": "^7.0.2"
```

Comment:
husky should probably stay in, because it's the level of the `linting`. @luke-song